### DISCUSSION TITLE
Admin interface - select format in editing component

### DISCUSSION THREAD

**Discussion by ronny35** [strapi/strapi#6091] [Legacy Topics]
Currently, when I edit a content with a component and this component has a relation field, the display of select has not a appropriate format (too small in my case).
Is it possible to have responsible display ?

![image](https://user-images.githubusercontent.com/15168987/81171477-bc987680-8f9c-11ea-9bbd-f13237d8fe20.png)

- Strapi version: Strapi v3.0.0-beta.18.7

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by lauriejim** [comment]
Hello! Sorry but I don't understand what you are talking about.
Can you please give us more details. And your screenshot is not really useful if you hive everything.

**Comment by ronny35** [comment]
Hello,

Sorry for this misunderstanding. The size of the select area is to small when the value is large.
Besides, it is very difficult to select the correct value.

<img width="557" alt="81171477-bc987680-8f9c-11ea-9bbd-f13237d8fe20" src="https://user-images.githubusercontent.com/15168987/82215161-40e0e700-9917-11ea-9307-8347c75d45ac.png">

**Comment by lauriejim** [comment]
I ping @soupette about this.

**Reply by soupette** [reply]
It has been fixed in the beta.20 so I think this discussion can be closed
